src/meds_torch/data/datamodule.py

```diff
@@ -107,10 +109,13 @@ def setup(self, stage: str | None = None) -> None:
             self.batch_size_per_device = self.hparams.cfg.dataloader.batch_size // self.trainer.world_size
 
         # load and split datasets only if not loaded already
-        self.data_train = get_dataset(self.cfg, split="train")
-        if stage != "train":  # TODO: remove this after we have more test data
+        if stage == "test":
```

Comment:
why use different names for stages vs. splits?

Reply:
The reason for this different naming scheme is to bridge the different naming schemes used in pytorch lightning vs meds-transform. But depending on how a user preprocessed data, any split name may be used.

I propose in the [data schema ](https://github.com/Oufattole/meds-torch/blob/a35a3a3917b0c7be7db7dc35c9a06010edbb45a3/src/meds_torch/configs/data/pytorch_dataset.yaml#L36) we have a split map added so users can define the proper splits they want:

```yaml
split_names
    train: train
    validate: tuning
    test: held_out
```